README.md

```diff
@@ -91,33 +94,40 @@ The following table shows which version of module documentation and declarations
 
 | Oracle Database  | Declarations | Documentation |
 | ---------------- | ------------ | ------------- |
-| 23ai | [mle-js@23.4.0][mle-js-types-234] for Oracle 23.4 <br/> [mle-js@23.3.0][mle-js-types-233] for Oracle 23.3 <br/> [mle-js@23.2.0][mle-js-types-232] for Oracle 23.2 - Free | **[mle-js (23ai)][mle-js]** <br/> [mle-js-oracledb (23ai)][mle-js-oracledb] <br/> [mle-js-bindings (23ai)][mle-js-bindings] <br/> [mle-js-plsqltypes (23ai)][mle-js-plsqltypes] <br/> [mle-js-fetch (23ai)][mle-js-fetch] <br/> [mle-encode-base64 (23ai)][mle-encode-base64]|
+| 23ai | [mle-js@23.5.0][mle-js-types-235] for Oracle 23.5 <br/> [mle-js@23.4.0][mle-js-types-234] for Oracle 23.4 <br/> [mle-js@23.3.0][mle-js-types-233] for Oracle 23.3 <br/> [mle-js@23.2.0][mle-js-types-232] for Oracle 23.2 - Free | **[mle-js (23ai)][mle-js]** <br/> [mle-js-oracledb (23ai)][mle-js-oracledb] <br/> [mle-js-bindings (23ai)][mle-js-bindings] <br/> [mle-js-plsqltypes (23ai)][mle-js-plsqltypes] <br/> [mle-js-fetch (23ai)][mle-js-fetch] <br/> [mle-encode-base64 (23ai)][mle-encode-base64]|
 | 21c | [mle-js@21.3.1][mle-js-types-213] | [mle-js-oracledb (21c)][mle-js-oracledb-21c] <br/> [mle-js-bindings (21c)][mle-js-bindings-21c] <br/> [mle-js-plsqltypes (21c)][mle-js-plsqltypes-21c] |
 
 ## Examples
 The following code snippet exemplifies the usage of some of these MLE modules combined.
 For additional examples, please check the module documentation pages or have a look at our [blog article][2].
 
 ```JavaScript
-// imports
-const oracledb = require('mle-js-oracledb');
-const bindings = require('mle-js-bindings');
-const plsqltypes = require("mle-js-plsqltypes");
-
-// Read a large number as an Oracle NUMBER from SQL and add another ORACLE NUMBER to it.
-// mle-js-oracledb is used for reading from SQL and mle-js-plsqltypes is used to construct the second Oracle NUMBER.
-const conn = oracledb.defaultConnection();
-const query = "SELECT 9007199254740992 AS n FROM dual";
-const options = { fetchInfo: { N: { type: oracledb.ORACLE_NUMBER } } };
-const queryResult = conn.execute(query, [], options);
-const OracleNumber = plsqltypes.OracleNumber;
-const result = queryResult.rows[0][0].add(new OracleNumber(7));
```

Comment:
quite sure this doesn't work - you need to set `outformat: oracledb.OUT_FORMAT_ARRAY` as part of the options.